js/scenes/AccountScreen/AccountScreenContainer.js

```diff
@@ -1,9 +1,16 @@
 import React from 'react';
+import { Image } from 'react-native';
 import AccountScreen from './AccountScreen'
 
 class AccountScreenContainer extends React.Component {
   static navigationOptions = {
-    title: 'Account Screen',
+    title: 'Account',
+    
+    tabBar:()=> ({
+      icon: (
+        <Image style={{height:30, width: 25}} source={require('../../../assets/images/accounticon.png')} />
```

Comment:
Just for sizing purposes, I changed it around a bit, but the width on this needs to stay at 25.